lib/kamal/cli/main.rb

```diff
@@ -185,11 +186,15 @@ def envify
       env_path          = ".env"
     end
 
-    File.write(env_path, ERB.new(File.read(env_template_path), trim_mode: "-").result, perm: 0600)
+    if File.exist?(env_template_path)
+      File.write(env_path, ERB.new(File.read(env_template_path), trim_mode: "-").result, perm: 0600)
 
-    unless options[:skip_push]
-      reload_envs
-      invoke "kamal:cli:env:push", options
+      unless options[:skip_push]
+        reload_envs
+        invoke "kamal:cli:env:push", options
+      end
+    else
+      puts "Skipping envify (no #{env_template_path} exist)"
```

Comment:
I'd keep using `say` as in other places.

```suggestion
      say "Skipping envify (no #{env_template_path} exist)", :magenta
```

Reply:
We're not in a context where we can use say.